From 94a23c3a5194d527aae993edbc68a50695a3f93e Mon Sep 17 00:00:00 2001
From: Colin Walters <walters@verbum.org>
Date: Tue, 20 Mar 2012 21:32:32 +0000
Subject: applet: Fix Makefile rule to remove libgnome-bluetooth-applet.la

1) We should be using install-exec-hook, not -data, since it's code
2) Use -f to avoid erroring out if for some reason it's not found
---
diff --git a/applet/Makefile-lib.am b/applet/Makefile-lib.am
index 375270c..251575d 100644
--- a/applet/Makefile-lib.am
+++ b/applet/Makefile-lib.am
@@ -13,8 +13,8 @@ libgnome_bluetooth_applet_la_LDFLAGS = -no-undefined
 
 AM_CFLAGS = -I$(srcdir) -I$(top_srcdir)/lib $(LIBGNOMEBT_CFLAGS) $(WARN_CFLAGS) $(DISABLE_DEPRECATED)
 
-install-data-hook:
-	rm $(DESTDIR)$(pkglibdir)/libgnome-bluetooth-applet.la
+install-exec-hook:
+	rm -f $(DESTDIR)$(pkglibdir)/libgnome-bluetooth-applet.la
 uninstall-hook:
 	rm -f $(DESTDIR)$(pkglibdir)/libgnome-bluetooth-applet.so*
 
--
cgit v0.9.0.2
